wcivf/apps/core/context_processors.py

```diff
@@ -2,9 +2,15 @@
 
 from .forms import PostcodeLookupForm
 
+# Remove this once we add all_settings to settings.CONTEXT_PROCESSORS
 def canonical_url(request):
     return {'CANONICAL_URL': settings.CANONICAL_URL}
 
+
+def all_settings(request):
+    return {x: getattr(s, x) for x in dir(s)}
```

Comment:
I guess you mean `settings` instead of `s` here?

Sorry for the drive-by comment, but this worries me a bit. e.g. it'll make `SECRET_KEY` and `DATABASES` available in the context, which people should never even think of using in a template, and could be leaked in DEBUG mode if you're using DjDT for example. (Not that a production site should be running with `DEBUG = True`, but these things happen sometimes.)

Also, using `dir` will mean that the magic methods of the `LazySettings` object get added to the context, which might be confusing... Personally, I prefer a safe-list approach like: https://github.com/DemocracyClub/yournextrepresentative/blob/161c2004a4b701a3429837ab4e9631b8f8640f4c/mysite/context_processors.py#L24-L47